src/pages/components/Navbar/Navbar.tsx

```diff
@@ -1,12 +1,16 @@
 import React, { useState } from 'react';
-import PropTypes from 'prop-types';
 import Image from 'next/image';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
 import { Link } from 'react-scroll';
 import SubMenu from './SubMenu';
 import MenuIcon from '../../assets/downchevron.svg';
 
-const Navbar = ({ to, transparent }) => {
+interface NavBarPropsInterface {
+  to: string;
+  transparent: boolean;
+}
+
+const Navbar = ({ to = '/', transparent }: NavBarPropsInterface) => {
```

Comment:
why are we providing the default value `'/'`? if we're providing a default value, then I think the `to` in `NavBarPropsInterface` should be optional